lib/image_optim/worker/jpegrecompress.rb

```diff
@@ -26,6 +26,19 @@ def self.init(image_optim, options = {})
         OptionHelpers.limit_with_range(v.to_i, 0...QUALITY_NAMES.length)
       end
 
+      METHOD_OPTION =
+      option(:method, 'ssim', 'Comparison Metric: '\
+          '`mpe` - Mean pixel error, '\
+          '`ssim` - Structural similarity, '\
+          '`ms-ssim` - Multi-scale structural similarity (slow!), '\
+          '`smallfry` - Linear-weighted BBCQ-like (may be patented)') do |v, opt_def|
+        unless %w[mpe ssim ms-ssim smallfry].include? v
+          warn "Unknown method for jpegrecompress: #{v}"
+          opt_def.default
```

Comment:
This will not be returned from the block. I would also suggest a spec for this, like in [pngquant_spec.rb#L35-L66](https://github.com/toy/image_optim/blob/129da7b25d45b0d57d9c9ac2045e9842810ef704/spec/image_optim/worker/pngquant_spec.rb#L35-L66).

Reply:
Okay, done! I added a spec file for jpegrecompress and I fixed the default return value when passing in an invalid method.